RELEASE.rst

```diff
@@ -0,0 +1,17 @@
+RELEASE_TYPE: patch
+
+This patch improves the quality of strategies inferred from Numpy dtypes:
+
+* Signed integer dtypes generated examples with some upper bits set to zero
```

Comment:
Just to check, did this mean we were never producing negative values? If so, ouch. That's pretty bad. Or was it just the high non-sign bits? Might be worth making that clearer here.